csharp/upgrades/57331d90231fc8247d5a5197f4ac45d502708c6a/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Add C#8 expression types
+compatibility: backwards
```

Comment:
The upgrade script need to be updated following my changes (sorry about not doing this myself).